examples/custom-node/src/primitives/tx.rs

```diff
@@ -17,9 +17,16 @@ use reth_codecs::{
 };
 use reth_ethereum::primitives::{serde_bincode_compat::SerdeBincodeCompat, InMemorySize};
 use reth_op::primitives::{Extended, SignedTransaction};
+use revm::context::TxEnv;
 use revm_primitives::{Address, Bytes};
 use serde::{Deserialize, Serialize};
 
+pub type CustomTransaction = ExtendedOpTxEnvelope<CustomTransactionEnvelope>;
+
+pub(crate) fn from(value: OpTxEnvelope) -> CustomTransaction {
+    Extended::BuiltIn(value)
+}
```

Comment:
we could just make this From impls in the reth-primitives-traits?

because the expectation is that these are always used as the `BuiltIn` variants?

Reply:
https://github.com/paradigmxyz/reth/pull/16366